EIPS/eip-7213.md

```diff
@@ -0,0 +1,251 @@
+---
+eip: [to be assigned]
```

Comment:
```suggestion
eip: 7619
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.